basyx.aasrepository/basyx.aasrepository-http/src/test/java/org/eclipse/digitaltwin/basyx/aasrepository/http/AasRepositoryHTTPSuite.java

```diff
@@ -283,6 +298,58 @@ public void paginationResult() throws FileNotFoundException, IOException, ParseE
 		BaSyxHttpTestUtils.assertSameJSONContent(getPaginatedAas1JSONString(), getJSONWithoutCursorInfo(response));
 	}
 
+	@Test
+	public void uploadThumbnailToShell() throws IOException {
+		createDummyAasOnServer(getAas1JSONString());
+		CloseableHttpResponse getThumbnailResponse = uploadThumbnail(dummyAasId);
+
+		assertEquals(HttpStatus.OK.value(), getThumbnailResponse.getCode());
+	}
+
+	@Test
+	public void getFile() throws FileNotFoundException, IOException, ParseException {
+		createDummyAasOnServer(getAas1JSONString());
+
+		byte[] expectedFile = readBytesFromClasspath(THUMBNAIL_FILE_PATH);
+
+		uploadThumbnail(dummyAasId);
+
+		CloseableHttpResponse response = BaSyxHttpTestUtils.executeGetOnURL(getThumbnailAccessURL(dummyAasId));
+		assertEquals(HttpStatus.OK.value(), response.getCode());
```

Comment:
this behaviour you have already tested with uploadThumbnailToShell

Reply:
The shell on the server has no thumbnail before, to test the Get, I need to set a thumbnail before.